From 9c9656594e47484283d545f2ebc396c58cb5a5aa Mon Sep 17 00:00:00 2001
From: Marcin Wojtas <mw@semihalf.com>
Date: Mon, 3 Jun 2019 17:07:47 +0200
Subject: [PATCH 280/386] net: mvpp2: notify when using packet recycling

Kernel space packet recycling feature is optional,
so notify when it is enabled.

Change-Id: Iac684a3e58244322b194825c9a2397ae6e3ab135
Signed-off-by: Marcin Wojtas <mw@semihalf.com>
Reviewed-on: https://sj1git1.cavium.com/10436
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Reviewed-by: Yan Markman <Yan.Markman@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 drivers/net/ethernet/marvell/mvpp2/mvpp2_main.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/mvpp2/mvpp2_main.c b/drivers/net/ethernet/marvell/mvpp2/mvpp2_main.c
index d2b9281ffa6c..7b029507fbe6 100644
--- a/drivers/net/ethernet/marvell/mvpp2/mvpp2_main.c
+++ b/drivers/net/ethernet/marvell/mvpp2/mvpp2_main.c
@@ -7469,8 +7469,11 @@ static int mvpp2_probe(struct platform_device *pdev)
 	/* Configure branch prediction switch */
 	if (priv->hw_version == MVPP21)
 		static_branch_enable(&mvpp21_variant);
-	if (recycle)
+	if (recycle) {
+		dev_info(&pdev->dev,
+			 "kernel space packet recycling feature enabled\n");
 		static_branch_enable(&mvpp2_recycle_ena);
+	}
 	/* else - keep the DEFINE_STATIC_KEY_FALSE */
 
 	/* Map DTS-active ports. Should be done before FIFO mvpp2_init */
-- 
2.17.1

